Sensitivity.jmd

```diff
@@ -0,0 +1,135 @@
+---
+weave_options:
+  error: false
+  echo: false
+---
+
+```julia
+import Pkg; Pkg.instantiate()
+
+Pkg.activate(".")
+using DrugResponseModel
+using ExponentialUtilities
+using Plots
+Plots.scalefontsizes(0.7)
+```
+To show the sensitivity of cell number difference predicted by combination.
+
+```julia
+concs, populations1, g1s1, g2s1 = load(189, 1);
+concs, populations2, g1s2, g2s2 = load(189, 2);
+concs, populations3, g1s3, g2s3 = load(189, 3);
+g0 = g1s1[1,1,1]+g2s1[1,1,1];
+
+g1avg = (g1s1 .+ g1s2 .+ g1s3) ./ 3; # pure data
+g2avg = (g2s1 .+ g2s2 .+ g2s3) ./ 3; # pure data
+popavg = g1avg .+ g2avg;
+IC50_lap, IC50_dox, IC50_gem, IC50_tax, IC50_pal = DrugResponseModel.find_IC(popavg, 0.5);
```

Comment:
This is a poor way to find the IC50 (it will essentially always be between the concentrations tested). Don't you have a better IC50 estimate from the parameter fits?

Reply:
Yes, they were different in two cases I guess. Should I use the IC50s that the model picked?

Reply:
I'm doing this, it takes a bit longer because I need to extrapolate the Hill effects based on the EC50.

Reply:
Oh.. I realized that I don't have to extrapolate... it is EC50! (please ignore me)